lib/anyplayer/players/rhythmbox.rb

```diff
@@ -46,14 +46,12 @@ def album
   end
 
   def launched?
-    %x(rhythmbox-client --no-start --print-playing 2>/dev/null).rstrip != ""
+    %x(rhythmbox-client --no-start --print-playing 2>/dev/null).rstrip != ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `%x` unless the command string contains backquotes.